app/code/Magento/InventoryExportStockApi/Api/ExportStockIndexDataInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryExportStockApi\Api;
+
+/**
+ * Interface for ExportStockIndexData which provides stock index export
+ * @api
+ */
+interface ExportStockIndexDataInterface
+{
+    /**
+     * Provides stock index export from inventory_stock_% table
+     *
+     * @param int $stockId
+     * @return array
+     */
+    public function execute(
+        int $stockId
```

Comment:
ExportStockIndexDataInterface
should be compatible with 
```
interface ExportStockSalableQtyInterface
{
    /**
     * Provides stock export data
     *
     * @param \Magento\Framework\Api\SearchCriteriaInterface $searchCriteria
     * @param string $websiteCode
     * @return \Magento\InventoryExportStockApi\Api\Data\ExportStockSalableQtySearchResultInterface
     */
    public function execute(
        \Magento\Framework\Api\SearchCriteriaInterface $searchCriteria,
        string $websiteCode
    ): \Magento\InventoryExportStockApi\Api\Data\ExportStockSalableQtySearchResultInterface;
}
```

because accepting stockId we can't define Manage Stock = false products, which we define by website assignment